example/src/modules/MultiStory/MultiStoryScreen.tsx

```diff
@@ -51,6 +58,18 @@ const MultiStoryScreen = () => {
               borderColor: Colors.lightGrey
             }
           }}
+          overlayViewPostion={'top'}
+          renderOverlayView={(item: StoryType) => (
+            <TouchableOpacity
```

Comment:
@nilesh-simform We should create this OverlayView as a component and use it here, instead of directly defining it. We can keep this component in this same file.